commerce_coordinator/apps/titan/clients.py

```diff
@@ -0,0 +1,103 @@
+"""
+API clients for Titan.
+"""
+
+import requests
+from celery.utils.log import get_task_logger
+from django.conf import settings
+from edx_rest_api_client.client import OAuthAPIClient
+
+# Use special Celery logger for tasks client calls.
+logger = get_task_logger(__name__)
+
+
+class TitanAPIClient():
```

Comment:
Does edX care about capitalization style?  Here API is all caps, but EcommerceApiClient has it camel case.  Ignore if we don't, just mentioning it in case we do!

Reply:
I was unsure, so I did a little research. Looks like the balance is in favor of all capital letters for acronyms:
* We use an internal library called `edx_rest_api_client`, which provides for us `OAuthAPIClient`
* We derive our views from Django REST Framework's `APIView`
* [PEP-8](https://peps.python.org/pep-0008/#descriptive-naming-styles) also recommends it: "When using acronyms in CapWords, capitalize all the letters of the acronym"

There is definitely a lot of variable usage across our projects though.